base_import_async/__manifest__.py

```diff
@@ -11,8 +11,15 @@
     "website": "https://github.com/OCA/queue",
     "category": "Generic Modules",
     "depends": ["base_import", "queue_job"],
-    "data": ["data/queue_job_function_data.xml", "views/base_import_async.xml"],
-    "qweb": ["static/src/xml/import.xml"],
+    "data": ["data/queue_job_function_data.xml"],
+    "assets": {
+        "web.assets_backend": [
+            "base_import_async/static/src/js/import.js",
+        ],
+        'web.assets_qweb': [
+            'base_import_async/static/src/xml/import.xml',
+        ],
+    },
     "installable": False,
```

Comment:
Thank you @RosSinch :)
It is still not installable :thinking:

Reply:
I've given up :)
But could you please explain to me why it should be installable in v15 if it wasn't in v14?

Reply:
`"installable": False,` was false in v14 because nobody did the migration work in this version, so it could not be used in v14. Perhaps odoo will still install it using the `-i` option, not sure, but this would be a kind of bug, the module would be shown as uninstallable in the UI.

As I understand you actually did the migration work so the module runs on v15, then the module should be flagged as installable, it would be a pity if the module cannot be installed if you made it installable, right? :)

Reply:
Yes, you are absolutely right.  It should be installable and it can, because the work is done for this.

Reply:
@guewen Is PR ready to merge?

Reply:
Build is :red_circle: 
Also, please, when you get back to it, squash your commits and rewrite the 1 remaining as `[15.0][MIG] base_import_async`